packages/playground/src/utils/contracts.ts

```diff
@@ -57,13 +57,21 @@ async function normalizeContract(
     consumption = 0;
   }
 
+  let farmId: number;
+  try {
+    farmId = (await grid.nodes.get({ id: c.nodeID })).farmId;
+  } catch {
+    farmId = 0;
+  }
```

Comment:
we already doing a call to get the node status, I believe we can get the farm id with the same call.

Reply:
done